### PR TITLE
Recentering

### PR DESCRIPTION
**Pull request by Humungus-Fungus**
Pressing space now centers cam on champ instead of the center of the map.
Pressing Y toggles this.

### CODE REVIEW

**Comment by noah1510** [comment]
You will have to reimplement this after #13 is merged. Both of you properly reformatted the player script but the other pr is for the whole project.

**Comment by noah1510** [comment]
Also all camera controls should be purely client only. Why is there server communication in camera movement? The clients don't need the server permission to move their camera where they want it to be.

**Comment by Humungus-Fungus** [comment]
Updated to work with #13 and now uses the names of the champion nodes to get their pid instead of asking the server.

**Review by noah1510 — CHANGES_REQUESTED** [review]
Remove Player.gd and ServerListener.gd since they no longer exist upstream.
Otherwise it looks good.

I will rework some of the centering logic in the future to support semi-locked camera controls. For that the center_cam will be reworked into a get_target_position which returns the follow target. Either you do that now or I will do that later.
Also the is_cam_centered option should be a config option but again, I can do that in the future.

**Comment by Humungus-Fungus** [comment]
Oops, didn't even notice the change in names. Reworked center_cam. Changed is_cam_centered into a config setting, now you have the option to change the center toggle through settings or with Y, both work.

File: Scripts/config.gd [hunk 1]
```diff
@@ -59,3 +60,8 @@ func set_min_zoom(new_value: float):
 func set_max_zoom(new_value: float):
 	max_zoom = new_value
 	camera_property_changed.emit()
+
+
+func set_cam_centered(new_value: float):
```
Comment (noah1510) [inline Scripts/config.gd]: should be bool

**Review by noah1510 — CHANGES_REQUESTED** [review]

**Comment by noah1510** [comment]
Besides the wrong type this is good to go

**Review by noah1510 — APPROVED** [review]

**Pull request merged**